### PR TITLE
style(datetimepicker): apply own style based on activeskin

### PR DESCRIPTION
**Pull request by joseglego**
# Changes
- Apply own style (based on activeskin) on the xdan datetimepicker
- This PR has a LOT of warnings, but are only 2 types:
- `Unexpected qualifying type selector (selector-no-qualifying-type) (scss-stylelint)`
- `Unexpected !important (declaration-no-important) (scss-stylelint)`

Both of them are required, why? I'm trying to overwrite the style of a third pack lib. This lib has everything written in this way. So, If I wanna be able to over-write it. I have to use the same or something more specific.

**Before:**
screen shot 2018-04-28 at 18 18 58

**After:**
screen shot 2018-04-28 at 18 44 07

### CODE REVIEW

File: app/assets/stylesheets/activeadmin_addons/inputs/date-time-picker.scss [hunk 1]
```diff
@@ -1,5 +1,301 @@
...
+        text-decoration: none;
+      }
+    }
+
+    .xdsoft_prev {
+      border-bottom: 5px solid $dt-select-bg !important;
+    }
+
+    .xdsoft_next {
+      border-top: 5px solid $dt-select-bg !important;
```
Comment (monkeyci) [inline app/assets/stylesheets/activeadmin_addons/inputs/date-time-picker.scss]: Unexpected !important                                                                                        declaration-no-important

File: app/assets/stylesheets/activeadmin_addons/inputs/date-time-picker.scss [hunk 2]
```diff
@@ -1,5 +1,301 @@
...
+
+      &:hover,
+      &:focus {
+        background: transparent;
+        text-decoration: none;
+      }
+    }
+
+    .xdsoft_prev {
+      border-bottom: 5px solid $dt-select-bg !important;
```
Comment (monkeyci) [inline app/assets/stylesheets/activeadmin_addons/inputs/date-time-picker.scss]: Unexpected !important                                                                                        declaration-no-important

File: app/assets/stylesheets/activeadmin_addons/inputs/date-time-picker.scss [hunk 3]
```diff
@@ -1,5 +1,301 @@
...
+      }
+    }
+  }
+
+  .xdsoft_timepicker {
+    .xdsoft_prev,
+    .xdsoft_next {
+      @extend .btn;
+      border-left: 5px solid transparent !important;
+      border-right: 5px solid transparent !important;
```
Comment (monkeyci) [inline app/assets/stylesheets/activeadmin_addons/inputs/date-time-picker.scss]: Unexpected !important                                                                                        declaration-no-important

File: app/assets/stylesheets/activeadmin_addons/inputs/date-time-picker.scss [hunk 4]
```diff
@@ -1,5 +1,301 @@
...
+        box-shadow: none;
+      }
+    }
+  }
+
+  .xdsoft_timepicker {
+    .xdsoft_prev,
+    .xdsoft_next {
+      @extend .btn;
+      border-left: 5px solid transparent !important;
```
Comment (monkeyci) [inline app/assets/stylesheets/activeadmin_addons/inputs/date-time-picker.scss]: Unexpected !important                                                                                        declaration-no-important

File: app/assets/stylesheets/activeadmin_addons/inputs/date-time-picker.scss [hunk 5]
```diff
@@ -1,5 +1,301 @@
...
...0iQWRvYmUgUGhvdG9zaG9wIENTNiAoV2luZG93cykiIHhtcE1NOkluc3RhbmNlSUQ9InhtcC5paWQ6QUExQUUzOTA0M0UyMTFFNDlBM0FFQTJENTExRDVBODYiIHhtcE1NOkRvY3VtZW50SUQ9InhtcC5kaWQ6QUExQUUzOTE0M0UyMTFFNDlBM0FFQTJENTExRDVBODYiPiA8eG1wTU06RGVyaXZlZEZyb20gc3RSZWY6aW5zdGFuY2VJRD0ieG1wLmlpZDpBQTFBRTM4RTQzRTIxMUU0OUEzQUVBMkQ1MTFENUE4NiIgc3RSZWY6ZG9jdW1lbnRJRD0ieG1wLmRpZDpBQTFBRTM4RjQzRTIxMUU0OUEzQUVBMkQ1MTFENUE4NiIvPiA8L3JkZjpEZXNjcmlwdGlvbj4gPC9yZGY6UkRGPiA8L3g6eG1wbWV0YT4gPD94cGFja2V0IGVuZD0iciI/Pp0VxGEAAAIASURBVHja7JrNSgMxEMebtgh+3MSLr1T1Xn2CHoSKB08+QmR8Bx9A8e7RixdB9CKCoNdexIugxFlJa7rNZneTbLIpM/CnNLsdMvNjM8l0mRCiQ9Ye61IKCAgZAUnH+mU3MMZaHYChBnJUDzWOFZdVfc5+ZFLbrWDeXPwbxIqrLLfaeS0hEBVGIRQCEiZoHQwtlGSByCCdYBl8g8egTTAWoKQMRBRBcZxYlhzhKegqMOageErsCHVkk3hXIFooDgHB1KkHIHVgzKB4ADJQ/A1jAFmAYhkQqA5TOBtocrKrgXwQA8gcFIuAIO8sQSA7hidvPwaQGZSaAYHOUWJABhWWw2EMIH9QagQERU4SArJXo0ZZL18uvaxejXt/Em8xjVBXmvFr1KVm/AJ10tRe2XnraNqaJvKE3KHuUbfK1E+VHB0q40/y3sdQSxY4FHWeKJCunP8UyDdqJZenT3ntVV5jIYCAh20vT7ioP8tpf6E2lfEMwERe+whV1MHjwZB7PBiCxcGQWwKZKD62lfGNnP/1poFAA60T7rF1UgcKd2id3KDeUS+oLWV8DfWAepOfq00CgQabi9zjcgJVYVD7PVzQUAUGAQkbNJTBICDhgwYTjDYD6XeW08ZKh+A4pYkzenOxXUbvZcWz7E8ykRMnIHGX1XPl+1m2vPYpL+2qdb8CDAARlKFEz/ZVkAAAAABJRU5ErkJggg==);
```
Comment (monkeyci) [inline app/assets/stylesheets/activeadmin_addons/inputs/date-time-picker.scss]: Expected quotes                                                                                              function-url-quotes

File: app/assets/stylesheets/activeadmin_addons/inputs/date-time-picker.scss [hunk 6]
```diff
@@ -1,5 +1,301 @@
...
+      }
+    }
+
+    .xdsoft_prev {
+      margin-left: 5px;
+      border-right: 5px solid white !important;
+    }
+
+    .xdsoft_next {
+      border-left: 5px solid white !important;
```
Comment (monkeyci) [inline app/assets/stylesheets/activeadmin_addons/inputs/date-time-picker.scss]: Unexpected named color "white"                                                                               color-named<br>Unexpected !important                                                                                        declaration-no-important

File: app/assets/stylesheets/activeadmin_addons/inputs/date-time-picker.scss [hunk 7]
```diff
@@ -1,5 +1,301 @@
...
+      &:hover,
+      &:focus {
+        background: transparent;
+        text-decoration: none;
+      }
+    }
+
+    .xdsoft_prev {
+      margin-left: 5px;
+      border-right: 5px solid white !important;
```
Comment (monkeyci) [inline app/assets/stylesheets/activeadmin_addons/inputs/date-time-picker.scss]: Unexpected named color "white"                                                                               color-named<br>Unexpected !important                                                                                        declaration-no-important

File: app/assets/stylesheets/activeadmin_addons/inputs/date-time-picker.scss [hunk 8]
```diff
@@ -1,5 +1,301 @@
...
+  background: transparent;
+}
+
+div.xdsoft_datetimepicker {
+  .xdsoft_datepicker.active {
+    .xdsoft_prev,
+    .xdsoft_next {
+      @extend .btn;
+      border-top: 5px solid transparent !important;
+      border-bottom: 5px solid transparent !important;
```
Comment (monkeyci) [inline app/assets/stylesheets/activeadmin_addons/inputs/date-time-picker.scss]: Unexpected !important                                                                                        declaration-no-important

File: app/assets/stylesheets/activeadmin_addons/inputs/date-time-picker.scss [hunk 9]
```diff
@@ -1,5 +1,301 @@
...
+  margin: 10px auto;
+  background: transparent;
+}
+
+div.xdsoft_datetimepicker {
+  .xdsoft_datepicker.active {
+    .xdsoft_prev,
+    .xdsoft_next {
+      @extend .btn;
+      border-top: 5px solid transparent !important;
```
Comment (monkeyci) [inline app/assets/stylesheets/activeadmin_addons/inputs/date-time-picker.scss]: Unexpected !important                                                                                        declaration-no-important

File: app/assets/stylesheets/activeadmin_addons/inputs/date-time-picker.scss [hunk 10]
```diff
@@ -1,5 +1,301 @@
...
+  color: white;
+  display: block;
+  width: 0;
+  height: 0;
+  margin: 10px auto;
+  background: transparent;
+}
+
+div.xdsoft_datetimepicker {
+  .xdsoft_datepicker.active {
```
Comment (monkeyci) [inline app/assets/stylesheets/activeadmin_addons/inputs/date-time-picker.scss]: Unexpected duplicate selector "div.xdsoft_datetimepicker .xdsoft_datepicker.active", first used at line 43   no-duplicate-selectors

File: app/assets/stylesheets/activeadmin_addons/inputs/date-time-picker.scss [hunk 11]
```diff
@@ -1,5 +1,295 @@
...
+    text-decoration: none !important;
+    color: $dt-calendar-color !important;
+  }
+
+  // Time. Selected
+  .xdsoft_timepicker .xdsoft_time_box>div>div.xdsoft_current {
+    box-shadow: none;
+    border-radius: 3px;
+    background-color: $dt-select-bg !important;
+    color: $dt-calendar-select-color !important;
```
Comment (monkeyci) [inline app/assets/stylesheets/activeadmin_addons/inputs/date-time-picker.scss]: Unexpected !important                                                                                         declaration-no-important

File: app/assets/stylesheets/activeadmin_addons/inputs/date-time-picker.scss [hunk 12]
```diff
@@ -1,5 +1,295 @@
...
+    background-color: $dt-calendar-hover-bg !important;
+    text-decoration: none !important;
+    color: $dt-calendar-color !important;
+  }
+
+  // Time. Selected
+  .xdsoft_timepicker .xdsoft_time_box>div>div.xdsoft_current {
+    box-shadow: none;
+    border-radius: 3px;
+    background-color: $dt-select-bg !important;
```
Comment (monkeyci) [inline app/assets/stylesheets/activeadmin_addons/inputs/date-time-picker.scss]: Unexpected !important                                                                                         declaration-no-important

File: app/assets/stylesheets/activeadmin_addons/inputs/date-time-picker.scss [hunk 13]
```diff
@@ -1,5 +1,295 @@
...
+  .xdsoft_timepicker .xdsoft_time_box >div >div {
+    background: $dt-bg;
+    color: $dt-color;
+  }
+
+  // Time. Hover
+  .xdsoft_timepicker .xdsoft_time_box >div >div:hover {
+    background-color: $dt-calendar-hover-bg !important;
+    text-decoration: none !important;
+    color: $dt-calendar-color !important;
```
Comment (monkeyci) [inline app/assets/stylesheets/activeadmin_addons/inputs/date-time-picker.scss]: Unexpected !important                                                                                         declaration-no-important

File: app/assets/stylesheets/activeadmin_addons/inputs/date-time-picker.scss [hunk 14]
```diff
@@ -1,5 +1,295 @@
...
+  // Time. Default
+  .xdsoft_timepicker .xdsoft_time_box >div >div {
+    background: $dt-bg;
+    color: $dt-color;
+  }
+
+  // Time. Hover
+  .xdsoft_timepicker .xdsoft_time_box >div >div:hover {
+    background-color: $dt-calendar-hover-bg !important;
+    text-decoration: none !important;
```
Comment (monkeyci) [inline app/assets/stylesheets/activeadmin_addons/inputs/date-time-picker.scss]: Unexpected !important                                                                                         declaration-no-important

File: app/assets/stylesheets/activeadmin_addons/inputs/date-time-picker.scss [hunk 15]
```diff
@@ -1,5 +1,295 @@
...
+
+  // Time. Default
+  .xdsoft_timepicker .xdsoft_time_box >div >div {
+    background: $dt-bg;
+    color: $dt-color;
+  }
+
+  // Time. Hover
+  .xdsoft_timepicker .xdsoft_time_box >div >div:hover {
+    background-color: $dt-calendar-hover-bg !important;
```
Comment (monkeyci) [inline app/assets/stylesheets/activeadmin_addons/inputs/date-time-picker.scss]: Unexpected !important                                                                                         declaration-no-important

File: app/assets/stylesheets/activeadmin_addons/inputs/date-time-picker.scss [hunk 16]
```diff
@@ -1,5 +1,295 @@
...
+        text-decoration: none;
+      }
+    }
+
+    .xdsoft_prev {
+      border-bottom: 5px solid $dt-select-bg !important;
+    }
+
+    .xdsoft_next {
+      border-top: 5px solid $dt-select-bg !important;
```
Comment (monkeyci) [inline app/assets/stylesheets/activeadmin_addons/inputs/date-time-picker.scss]: Unexpected !important                                                                                         declaration-no-important

File: app/assets/stylesheets/activeadmin_addons/inputs/date-time-picker.scss [hunk 17]
```diff
@@ -1,5 +1,295 @@
...
+
+      &:hover,
+      &:focus {
+        background: transparent;
+        text-decoration: none;
+      }
+    }
+
+    .xdsoft_prev {
+      border-bottom: 5px solid $dt-select-bg !important;
```
Comment (monkeyci) [inline app/assets/stylesheets/activeadmin_addons/inputs/date-time-picker.scss]: Unexpected !important                                                                                         declaration-no-important

File: app/assets/stylesheets/activeadmin_addons/inputs/date-time-picker.scss [hunk 18]
```diff
@@ -1,5 +1,295 @@
...
+      border-top-right-radius: 6px;
+    }
+  }
+
+  .xdsoft_timepicker {
+    .xdsoft_prev,
+    .xdsoft_next {
+      @extend .btn;
+      border-left: 5px solid transparent !important;
+      border-right: 5px solid transparent !important;
```
Comment (monkeyci) [inline app/assets/stylesheets/activeadmin_addons/inputs/date-time-picker.scss]: Unexpected !important                                                                                         declaration-no-important

File: app/assets/stylesheets/activeadmin_addons/inputs/date-time-picker.scss [hunk 19]
```diff
@@ -1,5 +1,295 @@
...
+      border-top-left-radius: 0;
+      border-top-right-radius: 6px;
+    }
+  }
+
+  .xdsoft_timepicker {
+    .xdsoft_prev,
+    .xdsoft_next {
+      @extend .btn;
+      border-left: 5px solid transparent !important;
```
Comment (monkeyci) [inline app/assets/stylesheets/activeadmin_addons/inputs/date-time-picker.scss]: Unexpected !important                                                                                         declaration-no-important

File: app/assets/stylesheets/activeadmin_addons/inputs/date-time-picker.scss [hunk 20]
```diff
@@ -1,5 +1,295 @@
...
+    }
+  }
+
+  .xdsoft_datepicker.active + .xdsoft_timepicker {
+    margin-top: 0;
+    margin-left: 0;
+
+    &::before {
+      content: '';
+      width:100%;
```
Comment (monkeyci) [inline app/assets/stylesheets/activeadmin_addons/inputs/date-time-picker.scss]: Expected single space after ":"                                                                               declaration-colon-space-after

File: app/assets/stylesheets/activeadmin_addons/inputs/date-time-picker.scss [hunk 21]
```diff
@@ -1,5 +1,295 @@
...
+    font-size: .8em;
+    padding-top: 1px;
+  }
+
+  .xdsoft_calendar td:hover {
+    background-color: transparent !important;
+
+    div {
+      background-color: $dt-calendar-hover-bg;
+      text-decoration: none !important;
```
Comment (monkeyci) [inline app/assets/stylesheets/activeadmin_addons/inputs/date-time-picker.scss]: Unexpected !important                                                                                         declaration-no-important

File: app/assets/stylesheets/activeadmin_addons/inputs/date-time-picker.scss [hunk 22]
```diff
@@ -1,5 +1,288 @@
...
+      color: $dt-calendar-color !important;
+    }
+
+    // Time. Selected
+    .xdsoft_time_box>div>div.xdsoft_current {
+      box-shadow: none;
+      border-radius: 3px;
+      background-color: $dt-select-bg !important;
+      color: $dt-calendar-select-color !important;
```
Comment (monkeyci) [inline app/assets/stylesheets/activeadmin_addons/inputs/date-time-picker.scss]: Unexpected !important                           declaration-no-important

File: app/assets/stylesheets/activeadmin_addons/inputs/date-time-picker.scss [hunk 23]
```diff
@@ -1,5 +1,288 @@
...
+      background-color: $dt-calendar-hover-bg !important;
+      text-decoration: none !important;
+      color: $dt-calendar-color !important;
+    }
+
+    // Time. Selected
+    .xdsoft_time_box>div>div.xdsoft_current {
+      box-shadow: none;
+      border-radius: 3px;
+      background-color: $dt-select-bg !important;
```
Comment (monkeyci) [inline app/assets/stylesheets/activeadmin_addons/inputs/date-time-picker.scss]: Unexpected !important                           declaration-no-important

File: app/assets/stylesheets/activeadmin_addons/inputs/date-time-picker.scss [hunk 24]
```diff
@@ -1,5 +1,288 @@
...
+    .xdsoft_time_box >div >div {
+      background: $dt-bg;
+      color: $dt-color;
+    }
+
+    // Time. Hover
+    .xdsoft_time_box >div >div:hover {
+      background-color: $dt-calendar-hover-bg !important;
+      text-decoration: none !important;
+      color: $dt-calendar-color !important;
```
Comment (monkeyci) [inline app/assets/stylesheets/activeadmin_addons/inputs/date-time-picker.scss]: Unexpected !important                           declaration-no-important

File: app/assets/stylesheets/activeadmin_addons/inputs/date-time-picker.scss [hunk 25]
```diff
@@ -1,5 +1,288 @@
...
+    // Time. Default
+    .xdsoft_time_box >div >div {
+      background: $dt-bg;
+      color: $dt-color;
+    }
+
+    // Time. Hover
+    .xdsoft_time_box >div >div:hover {
+      background-color: $dt-calendar-hover-bg !important;
+      text-decoration: none !important;
```
Comment (monkeyci) [inline app/assets/stylesheets/activeadmin_addons/inputs/date-time-picker.scss]: Unexpected !important                           declaration-no-important

File: app/assets/stylesheets/activeadmin_addons/inputs/date-time-picker.scss [hunk 26]
```diff
@@ -1,5 +1,288 @@
...
+
+    // Time. Default
+    .xdsoft_time_box >div >div {
+      background: $dt-bg;
+      color: $dt-color;
+    }
+
+    // Time. Hover
+    .xdsoft_time_box >div >div:hover {
+      background-color: $dt-calendar-hover-bg !important;
```
Comment (monkeyci) [inline app/assets/stylesheets/activeadmin_addons/inputs/date-time-picker.scss]: Unexpected !important                           declaration-no-important

File: app/assets/stylesheets/activeadmin_addons/inputs/date-time-picker.scss [hunk 27]
```diff
@@ -1,5 +1,288 @@
...
+        text-decoration: none;
+      }
+    }
+
+    .xdsoft_prev {
+      border-bottom: 5px solid $dt-select-bg !important;
+    }
+
+    .xdsoft_next {
+      border-top: 5px solid $dt-select-bg !important;
```
Comment (monkeyci) [inline app/assets/stylesheets/activeadmin_addons/inputs/date-time-picker.scss]: Unexpected !important                           declaration-no-important

File: app/assets/stylesheets/activeadmin_addons/inputs/date-time-picker.scss [hunk 28]
```diff
@@ -1,5 +1,288 @@
...
+
+      &:hover,
+      &:focus {
+        background: transparent;
+        text-decoration: none;
+      }
+    }
+
+    .xdsoft_prev {
+      border-bottom: 5px solid $dt-select-bg !important;
```
Comment (monkeyci) [inline app/assets/stylesheets/activeadmin_addons/inputs/date-time-picker.scss]: Unexpected !important                           declaration-no-important

File: app/assets/stylesheets/activeadmin_addons/inputs/date-time-picker.scss [hunk 29]
```diff
@@ -1,5 +1,288 @@
...
+      border-top-right-radius: 6px;
+    }
+  }
+
+  .xdsoft_timepicker {
+    .xdsoft_prev,
+    .xdsoft_next {
+      @extend .btn;
+      border-left: 5px solid transparent !important;
+      border-right: 5px solid transparent !important;
```
Comment (monkeyci) [inline app/assets/stylesheets/activeadmin_addons/inputs/date-time-picker.scss]: Unexpected !important                           declaration-no-important

File: app/assets/stylesheets/activeadmin_addons/inputs/date-time-picker.scss [hunk 30]
```diff
@@ -1,5 +1,288 @@
...
+      border-top-left-radius: 0;
+      border-top-right-radius: 6px;
+    }
+  }
+
+  .xdsoft_timepicker {
+    .xdsoft_prev,
+    .xdsoft_next {
+      @extend .btn;
+      border-left: 5px solid transparent !important;
```
Comment (monkeyci) [inline app/assets/stylesheets/activeadmin_addons/inputs/date-time-picker.scss]: Unexpected !important                           declaration-no-important

File: app/assets/stylesheets/activeadmin_addons/inputs/date-time-picker.scss [hunk 31]
```diff
@@ -1,5 +1,288 @@
...
+
+      &:hover {
+        background-color: transparent !important;
+
+          div {
+            background-color: $dt-calendar-hover-bg;
+            text-decoration: none !important;
+          }
+      }
+      div {
```
Comment (monkeyci) [inline app/assets/stylesheets/activeadmin_addons/inputs/date-time-picker.scss]: Expected empty line before rule                 rule-empty-line-before

File: app/assets/stylesheets/activeadmin_addons/inputs/date-time-picker.scss [hunk 32]
```diff
@@ -1,5 +1,289 @@
...
+
+    td {
+      height: 24px;
+
+      &:hover {
+        background-color: transparent !important;
+
+        div {
+          background-color: $dt-calendar-hover-bg;
+          text-decoration: none !important;
```
Comment (monkeyci) [inline app/assets/stylesheets/activeadmin_addons/inputs/date-time-picker.scss]: Unexpected !important                           declaration-no-important

File: app/assets/stylesheets/activeadmin_addons/inputs/date-time-picker.scss [hunk 33]
```diff
@@ -1,5 +1,289 @@
...
+      text-decoration: none !important;
+      color: $dt-calendar-color !important;
+    }
+
+    // Time. Selected
+    .xdsoft_time_box>div>div.xdsoft_current {
+      box-shadow: none;
+      border-radius: 3px;
+      background-color: $dt-select-bg !important;
+      color: $dt-calendar-select-color !important;
```
Comment (monkeyci) [inline app/assets/stylesheets/activeadmin_addons/inputs/date-time-picker.scss]: Unexpected !important                         declaration-no-important

File: app/assets/stylesheets/activeadmin_addons/inputs/date-time-picker.scss [hunk 34]
```diff
@@ -1,5 +1,289 @@
...
+      background-color: $dt-calendar-hover-bg !important;
+      text-decoration: none !important;
+      color: $dt-calendar-color !important;
+    }
+
+    // Time. Selected
+    .xdsoft_time_box>div>div.xdsoft_current {
+      box-shadow: none;
+      border-radius: 3px;
+      background-color: $dt-select-bg !important;
```
Comment (monkeyci) [inline app/assets/stylesheets/activeadmin_addons/inputs/date-time-picker.scss]: Unexpected !important                         declaration-no-important

File: app/assets/stylesheets/activeadmin_addons/inputs/date-time-picker.scss [hunk 35]
```diff
@@ -1,5 +1,289 @@
...
+    .xdsoft_time_box >div >div {
+      background: $dt-bg;
+      color: $dt-color;
+    }
+
+    // Time. Hover
+    .xdsoft_time_box >div >div:hover {
+      background-color: $dt-calendar-hover-bg !important;
+      text-decoration: none !important;
+      color: $dt-calendar-color !important;
```
Comment (monkeyci) [inline app/assets/stylesheets/activeadmin_addons/inputs/date-time-picker.scss]: Unexpected !important                         declaration-no-important

File: app/assets/stylesheets/activeadmin_addons/inputs/date-time-picker.scss [hunk 36]
```diff
@@ -1,5 +1,289 @@
...
+    // Time. Default
+    .xdsoft_time_box >div >div {
+      background: $dt-bg;
+      color: $dt-color;
+    }
+
+    // Time. Hover
+    .xdsoft_time_box >div >div:hover {
+      background-color: $dt-calendar-hover-bg !important;
+      text-decoration: none !important;
```
Comment (monkeyci) [inline app/assets/stylesheets/activeadmin_addons/inputs/date-time-picker.scss]: Unexpected !important                         declaration-no-important

File: app/assets/stylesheets/activeadmin_addons/inputs/date-time-picker.scss [hunk 37]
```diff
@@ -1,5 +1,289 @@
...
+
+    // Time. Default
+    .xdsoft_time_box >div >div {
+      background: $dt-bg;
+      color: $dt-color;
+    }
+
+    // Time. Hover
+    .xdsoft_time_box >div >div:hover {
+      background-color: $dt-calendar-hover-bg !important;
```
Comment (monkeyci) [inline app/assets/stylesheets/activeadmin_addons/inputs/date-time-picker.scss]: Unexpected !important                         declaration-no-important

File: app/assets/stylesheets/activeadmin_addons/inputs/date-time-picker.scss [hunk 38]
```diff
@@ -1,5 +1,289 @@
...
+        text-decoration: none;
+      }
+    }
+
+    .xdsoft_prev {
+      border-bottom: 5px solid $dt-select-bg !important;
+    }
+
+    .xdsoft_next {
+      border-top: 5px solid $dt-select-bg !important;
```
Comment (monkeyci) [inline app/assets/stylesheets/activeadmin_addons/inputs/date-time-picker.scss]: Unexpected !important                         declaration-no-important

File: app/assets/stylesheets/activeadmin_addons/inputs/date-time-picker.scss [hunk 39]
```diff
@@ -1,5 +1,289 @@
...
+
+      &:hover,
+      &:focus {
+        background: transparent;
+        text-decoration: none;
+      }
+    }
+
+    .xdsoft_prev {
+      border-bottom: 5px solid $dt-select-bg !important;
```
Comment (monkeyci) [inline app/assets/stylesheets/activeadmin_addons/inputs/date-time-picker.scss]: Unexpected !important                         declaration-no-important

File: app/assets/stylesheets/activeadmin_addons/inputs/date-time-picker.scss [hunk 40]
```diff
@@ -1,5 +1,289 @@
...
+      border-top-right-radius: 6px;
+    }
+  }
+
+  .xdsoft_timepicker {
+    .xdsoft_prev,
+    .xdsoft_next {
+      @extend .btn;
+      border-left: 5px solid transparent !important;
+      border-right: 5px solid transparent !important;
```
Comment (monkeyci) [inline app/assets/stylesheets/activeadmin_addons/inputs/date-time-picker.scss]: Unexpected !important                         declaration-no-important

File: app/assets/stylesheets/activeadmin_addons/inputs/date-time-picker.scss [hunk 41]
```diff
@@ -1,5 +1,289 @@
...
+      border-top-left-radius: 0;
+      border-top-right-radius: 6px;
+    }
+  }
+
+  .xdsoft_timepicker {
+    .xdsoft_prev,
+    .xdsoft_next {
+      @extend .btn;
+      border-left: 5px solid transparent !important;
```
Comment (monkeyci) [inline app/assets/stylesheets/activeadmin_addons/inputs/date-time-picker.scss]: Unexpected !important                         declaration-no-important

File: app/assets/stylesheets/activeadmin_addons/inputs/date-time-picker.scss [hunk 42]
```diff
@@ -1,5 +1,289 @@
...
+
+    td {
+      height: 24px;
+
+      &:hover {
+        background-color: transparent !important;
+
+        div {
+          background-color: $dt-calendar-hover-bg;
+          text-decoration: none !important;
```
Comment (monkeyci) [inline app/assets/stylesheets/activeadmin_addons/inputs/date-time-picker.scss]: Unexpected !important                         declaration-no-important

**Comment by difernandez** [comment]
Related to #146, closing this, as now we don't want to include many style changes in this gem that could depend on theme being used

**Pull request closed without merging**